packages/vx-geo/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Albers } from './components/Albers';
+export { default as Mercator } from './components/Mercator';
+export { default as Orthographic } from './components/Orthographic';
```

Comment:
@hshoff Yeah, sure. It think projections will be great. Is it a good idea to place the Projection.js  into this folder. There is no export for Projection.